server/chat-commands/room-settings.ts

```diff
@@ -7,6 +7,7 @@
  * @license MIT
  */
 import {Utils} from '../../lib/utils';
+import {ROOM_PERMISSIONS, GLOBAL_PERMISSIONS, GlobalPermission, RoomPermission} from '../user-groups';
```

Comment:
```suggestion
import type {RoomPermission} from '../user-groups';
```

Reply:
tbh we might want to export this type from `Users` so we don't have to import it all the time.